l10n_nl_bsn/README.rst

```diff
@@ -73,6 +73,8 @@ Contributors
 
 * Andrea Stirpe <a.stirpe@onestein.nl>
 
+Do not contact contributors directly about support or help with technical issues.
+
```

Comment:
the worst specimen of trolls won't be impeded by this to my experience, but then at least you can ignore them with good conscience ;-)